externs/es6.js

```diff
@@ -1050,13 +1050,13 @@ BigUint64Array.BYTES_PER_ELEMENT;
 
 /**
  * @param {string|!IArrayLike<bigint>|!Iterable<bigint>} source
- * @param {function(this:S, bigint): bigint=} mapFn
- * @param {S=} thisArg
+ * @param {function(this:S, ?, number): bigint=} opt_mapFn
```

Comment:
Our coding standards have changed so that we no longer want the `opt_` prefix on optional parameters.
So, rather than renaming the parameters here to add `opt_`, could you instead modify the older definitions to drop the `opt_` and match these newer definitions?

Please *don't* try to fix the whole file. That would just be noise.

Thanks

Reply:
Sure, updated those.